crates/spfs/src/runtime/spec_api_version.rs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Contributors to the SPK project.
+// SPDX-License-Identifier: Apache-2.0
+// https://github.com/spkenv/spk
+
+use serde::{Deserialize, Serialize};
+
+/// Supported spfs spec api versions
+#[derive(Debug, Deserialize, Serialize, Copy, Clone, Eq, PartialEq, strum::Display)]
+pub enum SpecApiVersion {
+    #[serde(
+        rename = "spfs/v0/livelayer",
+        alias = "v0/livelayer",
+        alias = "v0/layer"
```

Comment:
Feel free to remove these aliases unless you have a lot of these files lying around already - it seems better if we don't need to support these unqualified ones